internal/rbd/controllerserver.go

```diff
@@ -1226,6 +1226,17 @@ func (cs *ControllerServer) CreateSnapshot(
 		return nil, status.Error(codes.Internal, err.Error())
 	}
 
+	err = vol.Connect(cr)
```

Comment:
Should we add a comment explaining that `getImageInfo` updates the image size before creating a snapshot?